File: seed/php-sdk/pagination/no-custom-config/src/InlineUsers/InlineUsers/Types/ListUsersMixedTypePaginationResponse.php
<?php

namespace Seed\InlineUsers\InlineUsers\Types;

use Seed\Core\Json\JsonSerializableType;
use Seed\Core\Json\JsonProperty;

class ListUsersMixedTypePaginationResponse extends JsonSerializableType
{
    /**
     * @var string $next
     */
    #[JsonProperty('next')]
    public string $next;

    /**
     * @var Users $data
     */
    #[JsonProperty('data')]
    public Users $data;

    /**
     * @param array{
     *   next: string,
     *   data: Users,
     * } $values
     */
    public function __construct(
        array $values,
    )
    {
        $this->next = $values['next'];$this->data = $values['data'];
    }

    /**
     * @return string
     */
    public function __toString(): string {
        return $this->toJson();
    }
}

File: seed/ruby-sdk-v2/pagination/lib/seed/inline_users/inline_users/types/list_users_mixed_type_pagination_response.rb
# frozen_string_literal: true

module Seed
  module InlineUsers
    module InlineUsers
      module Types
        class ListUsersMixedTypePaginationResponse < Internal::Types::Model
          field :next_, -> { String }, optional: false, nullable: false, api_name: "next"
          field :data, -> { Seed::InlineUsers::InlineUsers::Types::Users }, optional: false, nullable: false
        end
      end
    end
  end
end
